simulation/models/control_room_columbia/reactor_physics/reactor_inventory.py

```diff
@@ -77,6 +79,11 @@ def run(delta):
     raw_level = mm_to_inches(calculate_level_cylinder(Vessel_Diameter,waterMass))
     rx_level_wr = raw_level-528.55
 
+    if rx_level_wr > 0:
```

Comment:
Good but probably easier and cleaner to write 
rx_level_nr = max(rx_level_wr,0)
i'll fix this when i pull it in